Tests/SwiftlyTests/HTTPClientTests.swift

```diff
@@ -45,35 +45,38 @@ final class HTTPClientTests: SwiftlyTests {
         XCTAssertTrue(exceptionThrown)
     }
 
-    func testGetFromGitHub() async throws {
-        // GIVEN: we have a swiftly http client with github capability
-        // WHEN: we ask for the first page of releases with page size 5
-        var releases = try await SwiftlyCore.httpClient.getReleases(page: 1, perPage: 5)
-        // THEN: we get five releases
-        XCTAssertEqual(5, releases.count)
+    func testGetMetdataFromSwiftOrg() async throws {
+        let supportedPlatforms = [
+            PlatformDefinition.macOS,
+            PlatformDefinition.ubuntu2204,
+            PlatformDefinition.ubuntu2004,
+            // PlatformDefinition.ubuntu1804,
```

Comment:
why commented out?

Reply:
Ubuntu 18.04 doesn't have toolchains available for the releases that we are testing here. I'll update the comment to explain.